ynr/apps/candidates/views/candidacies.py

```diff
@@ -58,10 +58,12 @@ def form_valid(self, form):
             person.not_standing.remove(self.ballot.election)
 
             if not membership_exists:
+                party = person.last_party()
```

Comment:
Not a line comment, but I noticed when reading the code that the LoggedAction created just above here is `ActionType.CANDIDACY_DELETE`. Could you fix that up while you're here?

Reply:
Good spot, do you think any of the LoggedAction objects in the live DB should be updated?

Reply:
No idea, I guess we can look for delete actions where the membership exits?

By the way, the reason I was looking in to this was because `last_party` is an interesting assumption we're making. I know it's not your code, but it's something we _might_ want to look in to at some point!

Reply:
That's an interesting point, as just because a person is standing again doesn't mean it is for the same party. In fact that made me have a look more in to where the `CandidacyView` is used because made me think of a scenario where its a first time candidate, where there is no `last_party` which would cause to an attribute error. But I think this view is only ever used when clicking the "Mark standing again" button, so perhaps we might want to change the name of this view also (at some point - not part of this PR)